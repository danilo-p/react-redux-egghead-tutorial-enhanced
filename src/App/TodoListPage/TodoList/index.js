import React from 'react';
import AddTodo from './AddTodo';
import VisibleTodoList from './VisibleTodoList';
import Footer from './Footer';

/**
 * Describes the todo list.
 *
 * @return {jsx} The TodoList template
 */
export default function TodoList() {
  return (
    <div>
      <AddTodo />
      <VisibleTodoList />
      <Footer />
    </div>
  );
}
