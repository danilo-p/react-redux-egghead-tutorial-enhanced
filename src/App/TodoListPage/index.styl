.TodoApp__Header
  margin-top: 1em !important;
  margin-bottom: 1em !important;