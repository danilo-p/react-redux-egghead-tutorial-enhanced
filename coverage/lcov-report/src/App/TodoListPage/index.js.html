<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/App/TodoListPage/index.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/App/TodoListPage</a> index.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>0/11</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>0/3</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25</td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React <span class="cstat-no" title="statement not covered" >from 'react';<span class="cstat-no" title="statement not covered" ></span></span>
import { Header, Container, Grid } from 'semantic-ui-react';
import TodoList from './TodoList';
import styles from './index.styl';
&nbsp;
/**
 * Describes the todo list page.
 *
 * @returns {jsx} The TodoListPage template
 */
export default function To<span class="cstat-no" title="statement not covered" >doListPage() {<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ></span></span></span></span></span></span></span></span></span>
  return (
    &lt;Container&gt;
      &lt;Grid centered&gt;
        &lt;Grid.Column computer="8" tablet="8" mobile="16"&gt;<span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ></span></span>
          &lt;Header as="h1" className={styles.TodoListPage__Header}&gt;
            A Simple Todo App
          &lt;/Header&gt;
          &lt;TodoList /&gt;
        &lt;/Grid.Column&gt;
      &lt;/Grid&gt;
    &lt;/Container&gt;
  );
}
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Jan 02 2018 00:10:32 GMT-0200 (-02)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
