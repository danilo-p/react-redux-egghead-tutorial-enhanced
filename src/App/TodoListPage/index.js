import React from 'react';
import { Header, Container, Grid } from 'semantic-ui-react';
import TodoList from './TodoList';
import styles from './index.styl';

/**
 * Describes the todo list page.
 *
 * @returns {jsx} The TodoListPage template
 */
export default function TodoListPage() {
  return (
    <Container>
      <Grid centered>
        <Grid.Column computer="8" tablet="8" mobile="16">
          <Header as="h1" className={styles.TodoListPage__Header}>
            A Simple Todo App
          </Header>
          <TodoList />
        </Grid.Column>
      </Grid>
    </Container>
  );
}
