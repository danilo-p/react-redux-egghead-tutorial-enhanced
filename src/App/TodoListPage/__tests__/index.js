import React from 'react';
import basicComponentTests from 'tests/basicComponentTests';
import TodoListPage from '../';

describe('TodoListPage', () => {
  basicComponentTests(<TodoListPage />);
});
