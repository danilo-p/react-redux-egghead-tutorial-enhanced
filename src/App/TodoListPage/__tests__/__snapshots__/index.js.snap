// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListPage should render the component 1`] = `
<Container>
  <Grid
    centered={true}
  >
    <GridColumn
      computer="8"
      mobile="16"
      tablet="8"
    >
      <Header
        as="h1"
        className="TodoListPage__Header"
      >
        A Simple Todo App
      </Header>
      <TodoList />
    </GridColumn>
  </Grid>
</Container>
`;
